stories/core_components/BarChart.tsx

```diff
@@ -31,6 +31,21 @@ import { withAxisNamePositionConfig, withAxisNameVisibilityConfig } from "../hoc
 const wrapperStyle = { width: 800, height: 400 };
 
 storiesOf("Core components/BarChart", module)
+    .add("one measure, one attribute", () =>
```

Comment:
added few bar chart storybooks to be able to compare with corresponding bullet chart